### PR TITLE
[wip - shezza has stuff to add]Adds Maid Cafe lavaland ruin. Moves some things to their own files.

### PR DESCRIPTION
**Pull request by silicons**
Requested by @Wilchenx 
image

Spawned mobs are instructed to put their duties above all else. Their current character slot is applied to their character, and then they are forcefully humanized and randomnamed.

	uniform = /obj/item/clothing/under/maid
	suit = /obj/item/clothing/suit/apron/purple_bartender
	back = /obj/item/storage/backpack/satchel/leather/withwallet
	gloves = /obj/item/clothing/gloves/fingerless
	shoes = /obj/item/clothing/shoes/laceup
	head = /obj/item/clothing/head/beret
	neck = /obj/item/clothing/neck/stripedredscarf
	ears = /obj/item/clothing/ears/headphones
	glasses = /obj/item/clothing/glasses/sunglasses
	id = /obj/item/card/id
	backpack_contents = list(/obj/item/storage/box/survival = 1, /obj/item/reagent_containers/spray/cleaner = 1, /obj/item/mop/advanced = 1)
	accessory = /obj/item/clothing/accessory/maidapron

### CODE REVIEW

**Comment by Repukan** [comment]
If this fetish shit is getting in, make them unable to leave their little hole.

**Comment by kevinz000** [comment]
@Repukan their flavortext implies that if they're going out to validhunt things admins can deal wit hthem

**Comment by kevinz000** [comment]
i'd rather not hardcode stuff to keep them in unless necessary

**Comment by Repukan** [comment]
Yeah that sure stopped syndicate scientists too right?

**Comment by kevinz000** [comment]
@Repukan and they can be jobbanned from all lavaland roles for all i care if they decide to abuse their role.
we haven't had an incident of those ones going out to kill the station/miners in a long time.

**Comment by Xhuis** [comment]
I'm indifferent to this either way, but this is like, literal, textbook, dictionary-definition fetish content. Usually those two words are used as buzzwords by people who aren't relevant to be angry at things they don't like, but it isn't really possible to deny that this is fetish content. O, Discordia.

**Comment by optimumtact** [comment]
how does someone make this pr without thinking about the consequences?

**Comment by MMMiracles** [comment]
don't you hate it when you're at work being someone's fetish when suddenly your entire workplace teleported to hell?

**Comment by ExcessiveUseOfCobblestone** [comment]
@Lexorion lol cucked if merged.

**Comment by Spaceinaba** [comment]
time to live out my sexual fantasies by serving meat to people as a caftgirl maid covered head to toe in blood and get blood all over the cafe and refuse to clean it up

**Comment by Spaceinaba** [comment]
then I'll serve the miners as desert

**Comment by Kromgar** [comment]
Please no fetish shit

**Comment by Qustinnus** [comment]
@kevinz000 you fell for the shezza bait lol

File: _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm [hunk 1]
```diff
@@ -0,0 +1,1585 @@
...
+/area/space)
+"ay" = (
+/obj/structure/window/reinforced{
+	dir = 1;
+	pixel_y = 1
+	},
+/obj/machinery/door/window/eastright{
+	dir = 8;
+	name = "Animal Pen";
+	req_access_txt = "0"
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm]: dv

File: _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm [hunk 2]
```diff
@@ -0,0 +1,1585 @@
...
+	},
+/turf/open/floor/grass,
+/area/space)
+"az" = (
+/obj/structure/window/reinforced{
+	dir = 1;
+	pixel_y = 1
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm]: is the 1 offset here needed?

File: _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm [hunk 3]
```diff
@@ -0,0 +1,1585 @@
...
+/turf/open/floor/plasteel/cafeteria,
+/area/space)
+"aX" = (
+/turf/open/floor/plasteel/cafeteria,
+/area/space)
+"aY" = (
+/obj/machinery/vending/wallmed{
+	name = "Emergency NanoMed";
+	pixel_y = 28;
+	req_access_txt = "0";
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_maid_cafe.dmm]: dv

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
It seems one of the fork contributors has accidentally PR'd to /tg/. Make sure to set your base branch to `Citadel-Station-13/Citadel-Station-13` next time

**Comment by PKPenguin321** [comment]
Remember when we added that bar sign and good threw a fit about how eventually somebody would try to add a maid cafe if it was merged and we didn't listen

**Comment by Iamgoofball** [comment]
# "its not a slippery slope" - goof deniers

**Comment by ExcessiveUseOfCobblestone** [comment]
@PKPenguin321 that was lex

**Comment by Iamgoofball** [comment]
i recall complaining about the sign too

**Comment by Repukan** [comment]
Slibbery slobbbb

:-DDDDDDDDDDDDD

Fuggggg :-DDDDD

**Comment by Slignerd** [comment]
Somehow I wouldn't mind this all that much on its own, and yet the fact that it pulls from the character slot rubs me the wrong way.

**Pull request closed without merging**